backend/Dockerfile.worker

```diff
@@ -3,7 +3,10 @@ FROM node:14-buster-slim as build
 WORKDIR /app
 
 COPY ./package* ./
-
+RUN apt-get update
+RUN apt-get -y install chromium
```

Comment:
Can you instead combine this with the `apt-get install -y` line later in the Dockerfile?

Reply:
Installation of `chromium` is required before `npm ci` otherwise the build fails when `puppeteer` is installed during `npm ci`

Reply:
I don't think installation of chromium is required before npm ci because you set PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true, so chromium is needed only on runtime

Reply:
hmmm let me try it out

Reply:
By meaning `later in the Dockerfile`, I assumed your suggestion is adding installation of chromium in [here](https://github.com/cisagov/crossfeed/pull/1226/files#diff-c122f7191ae217d53d59ac780bef85e8b21d6e330767d03a713e2ae5e58951adR22), but it actually failed with the following error.

```
[+] Building 39.0s (18/35)
 => [internal] load build definition from Dockerfile.worker                                                                                                                                                                                                                                                                        0.0s
 => => transferring dockerfile: 2.73kB                                                                                                                                                                                                                                                                                             0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                                                                                                                  0.0s
 => => transferring context: 34B                                                                                                                                                                                                                                                                                                   0.0s
 => [internal] load metadata for docker.io/library/node:14-buster-slim                                                                                                                                                                                                                                                             2.3s
 => [internal] load metadata for docker.io/library/golang:1.14.6-buster                                                                                                                                                                                                                                                            2.3s
 => [auth] library/golang:pull token for registry-1.docker.io                                                                                                                                                                                                                                                                      0.0s
 => [auth] library/node:pull token for registry-1.docker.io                                                                                                                                                                                                                                                                        0.0s
 => [internal] load build context                                                                                                                                                                                                                                                                                                  0.0s
 => => transferring context: 34.70kB                                                                                                                                                                                                                                                                                               0.0s
 => [stage-2  1/15] FROM docker.io/library/node:14-buster-slim@sha256:836375887d5ec180aeeeff3089560295fd1d2bd3644985e89ec6e7adf36a9d7e                                                                                                                                                                                             0.0s
 => [deps 1/7] FROM docker.io/library/golang:1.14.6-buster@sha256:b2a74bfbbe9b01536c2a3e0429060d90e37d99534f47ce48ecd38382a5c4c982                                                                                                                                                                                                 0.0s
 => CACHED [deps 2/7] WORKDIR /app                                                                                                                                                                                                                                                                                                 0.0s
 => CANCELED [deps 3/7] RUN apt-get update && apt-get install -y     wget     unzip     ca-certificates     chromium   && rm -rf /var/lib/apt/lists/*                                                                                                                                                                             36.6s
 => CACHED [stage-2  2/15] WORKDIR /app                                                                                                                                                                                                                                                                                            0.0s
 => CACHED [build 3/8] COPY ./package* ./                                                                                                                                                                                                                                                                                          0.0s
 => [build 4/8] RUN npm ci                                                                                                                                                                                                                                                                                                        18.8s
 => [build 5/8] COPY tsconfig.json ./tsconfig.json                                                                                                                                                                                                                                                                                 0.0s
 => [build 6/8] COPY webpack.worker.config.js ./webpack.worker.config.js                                                                                                                                                                                                                                                           0.0s
 => [build 7/8] COPY src ./src                                                                                                                                                                                                                                                                                                     0.1s
 => ERROR [build 8/8] RUN npx webpack --config webpack.worker.config.js                                                                                                                                                                                                                                                           17.3s
------
 > [build 8/8] RUN npx webpack --config webpack.worker.config.js:
```

Reply:
So I guess the only option is to add `apt-get install -y chromium` step right in front of `npx webpack`

Reply:
Sorry, you're right, I didn't notice the "FROM golang:1.14.6-buster as deps" section and the fact that the later apt-get command is part of a different docker stage, so the commands can't be combined. 

Thanks!